enterprise/ha/src/main/java/org/neo4j/kernel/ha/factory/HighlyAvailableEditionModule.java

```diff
@@ -161,6 +165,9 @@
 public class HighlyAvailableEditionModule
         extends EditionModule
 {
+    public static final Setting<Long> id_reuse_safe_zone_time =
+            setting( "dbms.id_reuse_safe_zone", Settings.DURATION, "1h" );
```

Comment:
Is this really the right place for this setting? What about `HaSettings`? You can even make it `@Internal` if you like.